zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/X509ZNodeGroupAclProvider.java

```diff
@@ -59,6 +59,11 @@
  *    "open read access": this feature concerns read access only. Add (world, anyone r) to all
  *          newly-written znodes whose path prefixes are given in the znode group acl config
  *          (comma-delimited, multiple such prefixes are possible).
+ *    "connection filtering": If the server is a dedicated server that only serves one resource,
+ *          the name of the resource is made the "namespace" of the server. The server will decline
+ *          the connection requests from client who belongs to a domain that does not match with the
+ *          server's namespace, only allow connection to be established with client belong to the
+ *          namespace domain and skip any further ACL check when processing client requests.
```

Comment:
"skip any further ACL check" - confusing. Does this mean that individual ZNodes will not be getting any ACL?